hack/build-clusteradm-image.sh

```diff
@@ -22,7 +22,7 @@
 set -e
 
 clusteradm_version="" # ==> latest
-clusteradm_folder=clusteradm
+clusteradm_folder="/tmp/clusteradm"
```

Comment:
What makes `/tmp/clusteradm` better than `./clusteradm`? I can see one downside to using `/tmp`: it is less protected by the filesystem.

Reply:
Are you saying that the binary should go to `/tmp` rather than `.` because it is annoying to have `git` complain about the binary not being staged for commit? That can easily be avoided by adding `clusteradm` to `.gitignore`.

Reply:
No, I'm concerned that somebody already has a local `./clusteradm` folder with stuff in it that he/she does not want to be overwritten and deleted at the end. I will change to use tempdir when I have a chance.

Reply:
The scripts in `hack/` are intended for use by contributors. These are people who have checked out the kubestellar repository. The normal expectation is that the current working directory is `kubestellar` (or possibly a subdirectory), unless the contributor is explicitly told otherwise. This script is used in the process of building and publishing KubeStellar, so I do not think that we need to worry about it being used with an arbitrary current working directory.